Jellyfin.Server/Jellyfin.Server.csproj

```diff
@@ -39,6 +39,10 @@
     <CodeAnalysisRuleSet>../jellyfin.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputPath>C:\Program Files\Jellyfin\Server</OutputPath>
```

Comment:
I think this might be what's making it so hard for me to test - Jellyfin.Server.dll is output to a different directory from everything else....

Reply:
i've removed it from the latest update.